src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -1700,6 +1769,32 @@ def set_up_defender(self, mc: ManagedCluster) -> ManagedCluster:
             mc.security_profile.defender = defender
 
         return mc
+    
+    def set_up_azure_monitor_profile(self, mc: ManagedCluster) -> ManagedCluster:
```

Comment:
This (AKSPreviewManagedClusterCreateDecorator/set_up_xxx) is used for `aks create` code path. But no parameter declaration for those options.

Reply:
removed, initially I was planning on updating the update path as well but we decided to change that later on. Deleted left over code.